ietf/templates/meeting/session_details_panel.html

```diff
@@ -320,51 +320,58 @@ <h3 class="mt-4">Notes and recordings</h3>
                 </tr>
             {% endif %}
             {# Recordings #}
-            {% if session.has_recordings %}
-                {% with session.recordings as recordings %}
-                    {% if recordings %}
-                        {# There's no guaranteed order, so this is a bit messy: #}
-                        {# First, the audio recordings, if any #}
-                        {% for r in recordings %}
-                            {% if r.get_href and 'audio' in r.get_href %}
-                                <tr>
-                                    <td>
-                                        <a href="{{ r.get_href }}"><i class="bi bi-file-play"></i> {{ r.title }}</a>
-                                    </td>
-                                </tr>
-                            {% endif %}
-                        {% endfor %}
-                        {# Then the youtube recordings #}
-                        {% for r in recordings %}
-                            {% if r.get_href and 'youtu' in r.get_href %}
-                                <tr>
-                                    <td>
-                                        <a href="{{ r.get_href }}"><i class="bi bi-file-slides"></i> {{ r.title }}</a>
-                                    </td>
-                                </tr>
-                            {% endif %}
-                        {% endfor %}
-                        {# Finally, any other recordings #}
-                        {% for r in recordings %}
-                            {% if r.get_href and not 'audio' in r.get_href and not 'youtu' in r.get_href %}
-                                <tr>
-                                    <td>
-                                        <a href="{{ r.get_href }}"><i class="bi bi-file-play"></i> {{ r.title }}</a>
-                                    </td>
-                                </tr>
-                            {% endif %}
-                        {% endfor %}
-                    {% endif %}
-                {% endwith %}
-                {% if session.video_stream_url %}
+            {% with session.recordings as recordings %}
```

Comment:
Note that I just removed the [`session.has_recordings`](https://github.com/ietf-tools/datatracker/blob/main/ietf/meeting/models.py#L391) check and looped around any associated recordings. Is this safe to do?

Reply:
Should be, but you could always prove it with a test.

Reply:
Test added